networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -213,13 +214,39 @@ def get_azs(self) -> List[conf.AvailabilityZone]:
             azs.append(conf.AvailabilityZone(name=site.slug, suffix=suffix, number=number))
         return sorted(azs, key=attrgetter('number'))
 
-    def get_cloud_vrfs(self) -> List[conf.VRF]:
+    def get_address_scopes(self, address_scope_vrf_maps_path: List[Path]) -> Dict[str, List[str]]:
+        result = defaultdict(list)
+        for p in address_scope_vrf_maps_path:
+            with p.open('r') as f:
+                asvm = yaml.safe_load(f)
+                for k, values in asvm.items():
+                    if not isinstance(k, str) or not k.endswith('address_scopes'):
+                        continue
```

Comment:
Why do the search for `.*address_scopes`? We have an expected format for this file. I'd check for the key `global_address_scopes` and if it's not there raise an error.

Reply:
Same as in above comment. helm-charts allow 2 places to define address scopes, once with `global_address_scopes` once with `local_address_scopes`. To also support if somebody just wants a YAML file with the key `address_scopes` I implemented it that way.

Reply:
Hmm. I see. Should we limit this to the two options then? I'm just thinking about somebody throwing an `blabla_address_scopes` in there and it breaks our config gen.